applications/ContactStructuralMechanicsApplication/custom_external_libraries/bprinter/README.md

```diff
@@ -0,0 +1,69 @@
+#BPrinter
+
```

Comment:
we agreed that this library will be removed no?

Reply:
Yes, but in a future PR, there is a Pending PR, but pooyan  told me to rework from the scratch for the logging macros